include/PlayPos.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * PlayPos.h - declaration of class PlayPos, a TimePos with additional features useful for Timelines
+ *
```

Comment:
I'm not sure `TimePos` can easily be refactored to store frames as its smallest unit of time, unless we allow for fractional frames. If the number of frames in a bar is not a whole number, then it will have to be rounded. That's fine, but if the bpm is changed, then every single TimePos will have to be updated at once so that the number of frames matches the correct tick amount. I feel like a better long-term solution would be to allow float tick values, but that's just me.

But I agree, the jumped variable can probably be combined with the `positionChanged` signal.

Reply:
What about just storing the frame position directly and still removing `PlayPos`?